examples/serial/candle-nvidia/Dockerfile

```diff
@@ -0,0 +1,124 @@
+# the user to launch an interactive container and resume building. From their 
+# image documentation:
+# 
+#    To complete CANDLE setup, execute source /opt/candle_setup.sh Next, copy 
+#    the script located in ‘/opt/testscript.sh’ to some other directory inside 
+#    the container and execute it. This script clones the CANDLE Benchmark and 
+#    Tutorials suites and downloads the appropriate data to run a CANDLE 
+#    hyperparameter optimization sweep. 
+#
+# We prefer to encapsulate the entire build process within the Dockerfile. 
+# Therefore the remainder of this Dockerfile installs and configures the various
+# dependancies required to run CANDLE (the steps that the user was expected to 
+# complete during runtime).
+#
+# The resulting container will enable end users to run CANDLE on a single node 
+# with GPU(s). Multinode runs via srun fail, it is unclear whether or not CANDLE
+# is designed to run in parallel; the examples appear to be executed in serial.
+COPY requirements.txt /opt
+
+WORKDIR /opt
+RUN pip install --upgrade pip && \
+    pip install -r requirements.txt
+
+ENV PATH="/usr/bin:${PATH}"
+ENV BUILD_DIR /opt
+
+RUN rm -rf swift-t 2> /dev/null
+
+RUN apt-get update -y && \
+    apt-get install -y --no-install-recommends \
+    autoconf \
+    automake \
+    default-jdk \
+    git \
+    ant \
+    swig \
+    zsh \
+    tcl \
+    tcl-dev \
+    python-tk
+
+RUN rm -rf /var/lib/apt/lists/* && \
+    wget http://swift-lang.github.io/swift-t-downloads/1.4/swift-t-1.4.tar.gz && \
+    tar -zxvf swift-t-1.4.tar.gz && \
+    $BUILD_DIR/swift-t-1.4/dev/build/init-settings.sh && \
+    sed -i  's@^export SWIFT_T_PREFIX=.*@export SWIFT_T_PREFIX=/opt/swift-t@' ./swift-t-1.4/dev/build/swift-t-settings.sh && \
+    sed -i  's@.*ENABLE_PYTHON=1@ENABLE_PYTHON=1@' ./swift-t-1.4/dev/build/swift-t-settings.sh && \
+    sed -i  's@.*ENABLE_R=1@ENABLE_R=1@' ./swift-t-1.4/dev/build/swift-t-settings.sh && \
+    sed -i  's@.*R_INSTALL=.*@R_INSTALL=/usr/share/R/@' ./swift-t-1.4/dev/build/swift-t-settings.sh && \
+    $BUILD_DIR/swift-t-1.4/dev/build/build-all.sh
+
+ENV TURBINE_PATH="${BUILD_DIR}/swift-t/turbine"
+RUN chmod +x $TURBINE_PATH/bin/turbine && \
+    sed -i  's@\${TURBINE_LAUNCHER} \${OPTIONS} \${=VALGRIND} @\${TURBINE_LAUNCHER} \${OPTIONS} \${=VALGRIND} \${TURBINE_HOME}/bin/gpurunscript.sh @' $TURBINE_PATH/bin/turbine && \
+    cp /gpurunscript.sh $TURBINE_PATH/bin && \
+    chmod +x $TURBINE_PATH/bin/gpurunscript.sh
+
+ENV PATH="${BUILD_DIR}/swift-t/stc/bin/:${TURBINE_PATH}/bin:${PATH}"
+ENV TURBINE_LAUNCH_OPTIONS --allow-run-as-root
+
+# Install our forked EQ-R. At the time of this writing the upstream EQ-R 
+# Makefile was broken (https://github.com/emews/EQ-R/pull/6). Our forked version
+# addresses the issue and ensures that this Dockerfile will continue to build
+# properly going forward, regardless of upstream changes.
+RUN git clone https://github.com/j-ogas/EQ-R.git
```

Comment:
Let's put the fork in the `hpc` namespace (rather than `j-ogas`) and check out a specific commit hash.

Did you submit an upstream PR with your patches?